core/templates/dev/head/domain/collection/CollectionNodeObjectFactory.js

```diff
@@ -17,20 +17,30 @@
  * collection node domain objects.
  */
 
-// TODO(bhenning): Implement validation functions and related tests.
+// TODO(bhenning): This should be provided by the backend.
```

Comment:
As discussed offline, I kept this and added a context string to guard against developers accidentally changing the value. Also, there are frontend tests which depend on this constant staying as-is. Finally, I filed #1695 to track work done to finish what was started here to better link the backend constants with frontend usage.